action.yml

```
@@ -15,7 +15,7 @@ runs:
   steps:
     # NAMESPACE LAB VERSION
     - name: Checkout Repository
-      uses: namespacelabs/nscloud-checkout-action@v2
+      uses: namespacelabs/nscloud-checkout-action@v5.0.3
```

don't pin to latest, pin to major only``
```suggestion
      uses: namespacelabs/nscloud-checkout-action@v5
```

---

hiya, have updated the versionings to latest major